docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -6,14 +6,245 @@ The parent key for all of the following parameters is `openshift4_local_storage`
+
+// don't look at this too closely, except when it's rendered -SG,2021-07-06.
+[cols=".^9,.^1,.^11",grid="none"]
+|===
+a|
+.Config in hierarchy
+[source,yaml]
+----
+storage_class_devices:
+  localstorage:
+    volumeMode: Block
+    devicePaths:
+      - /dev/vdb
+----
+a|pass:[<span style="font-size: 200%">&#8658;</span>]
+a|
+.Resulting `LocalVolume` resource
+[source,yaml]
+----
+apiVersion:
+kind: LocalVolume
+metdata:
+  name: ... # omitted
+spec:
+  storageClassDevices:
+    - storageClassName: localstorage
+      volumeMode: Block
+      devicePaths:
+        - /dev/vdb
+  # remaining spec omitted
+----
+|===
+
+=== Key `restricted_to`
+
+[horizontal]
+type:: dict
+keys:: namespace labels
+values:: one of
+* empty dict (`{}`)
+* dict with key `values`
+
+If this key is present, the component creates an Espejo `SyncConfig` to restrict the use of the storage classes created by the `LocalVolume` resource.
+
+This restriction is implemented with `ResourceQuota` resources which give a quota of 0 PVCs for the storage classes.
+With this `ResourceQuota` configuration, the resource quota must be configured in all namespaces other than the ones that are allowed to use the storage class.
+Therefore, the component must to invert the given restrictions to correctly restrict usage of the storage class to the specified namespaces.
+
+The content of the key is expected to be a dict, with keys in the dict corresponding to labels on namespaces.
+
+The values of the dict can be
+
+* Empty dicts (`{}`).
+In this case, the component constructs the following namespace selector `matchExpressions` entry:
++
+[source,yaml]
+----
+namespaceSelector:
+  labelSelector:
+    matchExpressions:
+      - key: <KEY> <1>
+        operator: DoesNotExist <2>
+----
+<1> The key in the `restricted_to` dict is used as value for field `key` in the match expression
+<2> The `ResourceQuota` needs to be present in all namespaces which don't have the label `<KEY>`.
+
+* Dicts with key `values`.
+In this case, the component constructs the following namespace selector `matchExpressions` entry:
++
+[source,yaml]
+----
+namespaceSelector:
+  labelSelector:
+    matchExpressions:
+      - key: <KEY> <1>
+        operator: NotIn <2>
+        values: < restricted_to[KEY].values > <3>
+----
+<1> The key in the `restricted_to` dict is used as value for field `key` in the match expression
+<2> The `ResourceQuota` needs to be present in all namespaces which don't have the label `<KEY>`.
+<3> The contents of field `values`.
+
+
+== Example configuration
+
+This example configuration shows how to present device `/dev/vdb` on all nodes with label `node-role.kubernetes.io/storage` as a PV with `volumeMode=Block` and storage class `localblock-storage`.
+
+The example restricts the use of the resulting storage class `localblock-storage` to namespaces labelled with `syn.tools/component=rook-ceph`.
+
+[source,yaml]
```

Comment:
I think this Rook example could be best described in a how-to page?

Reply:
There's a section `== Example` in `references/parameters.adoc` in the component template. As far as I understand, the idea of that section is to give the reader of the parameters reference documentation an impression of how the config might look in one piece.

Reply:
Yes, but currently it looks like a complete example that is almost as big as the reference itself :) I guess context matters a bit.
It's your choice, I don't have a hard feeling here.